.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -73,4 +73,7 @@ tracker: https://github.com/YOURORG/your_repo/issues
 license:
   url: http://creativecommons.org/licenses/by/3.0/
   label: CC-BY
+usages:
```

Comment:
"usages" sounds weird. could you say "examples of use" or does the keyword there have to be one word? if it has to be one word, maybe "uses"? or just "usage"

Reply:
this is not up to me - this is all official obo metadata! its all decided centrally there!

Reply:
ah, ok, if "usages" is the standard label here, then so be it.